apps/check-processing-time/src/check_processing_time.py

```diff
@@ -0,0 +1,14 @@
+import json
+
+
+def get_time_from_attempts(attempts):
+    attempts.sort(key=lambda attempt: attempt['StartedAt'])
+    final_attempt = attempts[-1]
+    return final_attempt['StoppedAt'] - final_attempt['StartedAt']
```

Comment:
I think `StartedAt` and `StoppedAt` are in milliseconds, so we may need to divide by 1,000 here
https://docs.aws.amazon.com/batch/latest/APIReference/API_AttemptDetail.html#Batch-Type-AttemptDetail-startedAt

Reply:
Ah, I meant to double-check those units. Thanks for the link! I was searching for that documentation but couldn't find it.

Reply:
I decided not to do any rounding because it could result in failed jobs with < 500 ms runtimes being shown as having runtime `0`, which should indicate that no processing was attempted. But I could instead always round up, if we'd rather `processing_time_in_seconds` be an integer instead of a float.